Moviebuff/Views/WatchlistView.swift

```diff
@@ -0,0 +1,81 @@
+
+import SwiftUI
+
+import SwiftUI
+import SDWebImageSwiftUI
+
+struct WatchlistView: View {
+    @ObservedObject var viewModel=MovieViewModel()
+    var body: some View {
+        NavigationView{
+            ZStack{
+                if let movies=viewModel.movies{
+                    MovListView(movies: movies, viewModel:viewModel)
+                }else{
+                    LoadingView()
+                }
+            }.navigationTitle("Watchlist")
+            }
+        .onAppear(){
+            viewModel.fetchMovieData(with: "/discover/movie?primary_release_date.gte=2022-10-15&primary_release_date.lte=2022-11-04")
+        }
+    }
+}
+
+struct MovieView: View{
+    var movie: Movie
+    var body: some View{
+  
+                HStack{
+                    Image(systemName: "circle")
+                    WebImage(url:URL(string:
+                "https://image.tmdb.org/t/p/w500/\(movie.poster_path)"))
```

Comment:
I was going to suggest extracting this into movie, but it looks like you already have but just didn't use it. I always like to keep constants and things like that out of views in general :).